README.md

```diff
@@ -30,6 +32,38 @@ It allows us to use features of `morley` (such as testing engine) with this cont
 Tests are implemented in the same Haskell package.
 Please refer to the [`haskell/`](/haskell/) directory for details.
 
+## Gas / Transaction costs
+
+The tables below show the gas and transaction costs of both versions (FA1.2 and FA2) of
+the stablecoin contract v1.3.0 in both Carthagenet and Delphinet.
+
```

Comment:
I would also provide links to better-call.dev or at least addresses of the contracts that were used for these experiments, maybe some people will want to see details (e. g. which transactions were made or how much storage was consumed).

Reply:
Good idea, done!